### PR TITLE
clean up text

### PR DESCRIPTION
**Pull request by edwardhartnett**
The explanatory text is very wordy. Some suggestions:
* Remove unneeded phrases like "Nevertheless", "Therefore", etc. They are appropriate for spoken English but not technical documentation.
* Long paragraphs need to be broken up.
* Long sentences need to be broken up.
* Better use of bullet and numbered lists would help.

In this PR I take an initial stab are removing unneeded text and adding paragraph and sentence breaks. Much more can be done but I think these changes make the documentation more readable to a newbie like myself.

Part of #397

### CODE REVIEW

**Comment by jbathegit** [comment]
I don't necessarily agree with all of the proposed changes, but then again I'm also not a newbie, so I'll defer to your judgement if you feel that this helps make everything clearer to a reader such as yourself ;-)

File: docs/dx_tables.md [hunk 1]
```diff
@@ -1,549 +1,74 @@
 
 # DX BUFR Tables
-@brief Description and format of DX BUFR tables for use with the library.
 @anchor dfbftab
```
Comment (jbathegit) [inline docs/dx_tables.md:5]: The \@brief is there for a reason.  If you go to the main documentation page (such as https://emc.ncep.noaa.gov/emc/pages/infrastructure/bufrlib_jeff/html/index.html which is the last time I downloaded and looked at the docs artifact from a developer CI run ;-), and if you then click on the "Related Pages" tab, you'll see the contents of the \@brief tag next to that page name.  Otherwise, if there's no \@brief in the .md file, then that part is blank with no description of what's actually in that page.  So it's a Doxygen thing, and I think it needs to be there.

File: docs/dx_tables.md [hunk 2]
```diff
@@ -1,549 +1,74 @@
...
 call to openbf(). For all other cases, DX table information must be pre-defined and made available
-to the software via call argument LUNDX during the call to openbf(). In the case of an existing
-BUFR file, the DX tables information may be embedded within the first few BUFR messages of the
+to the software via call argument LUNDX during the call to openbf(). The DX tables information may be embedded within the first few BUFR messages of the
 file itself. Otherwise, a separate ASCII text file containing the necessary DX tables information
-must be supplied, such as the example shown above. It is extremely important that any such file
-not only be syntactically correct but also complete, in the sense that all necessary mnemonics
+must be supplied, such as the example shown above. Such files
+must be syntactically correct and also complete, in the sense that all necessary mnemonics
 must exist and be fully-defined.
```
Comment (jbathegit) [inline docs/dx_tables.md]: change to "example shown below", since you've now moved the example to the end of the file ;-)

File: docs/dx_tables.md [hunk 3]
```diff
@@ -1,549 +1,74 @@
...
 
-Before continuing on let's recall
-that a DX BUFR tables file consists of three distinct sections.  Each section contains one or more
+Mnemonics never
+themselves appear within actual BUFR messages.
+Their only purpose is to make it easier for users to interact with the software by
+providing descriptive names to represent individual data values. They are a more intuitive than
+FXY numbers (described below). FXY numers are the prescribed method within the BUFR messages.
+
+## DX BUFR Tables File
```
Comment (jbathegit) [inline docs/dx_tables.md:54]: How about changing the last 2 sentences to "They are more intuitive than
FXY numbers (described below), which are the prescribed method within actual BUFR messages."?

Otherwise, if you think that's too wordy or unclear, then please at least remove the superfluous "a" near the start of the 1st sentence, and correct the spelling of numers to numbers in the 2nd sentence.

File: docs/dx_tables.md (hunk 2 above)
Comment (edwardhartnett) [reply docs/dx_tables.md]: Done.

File: docs/dx_tables.md (hunk 3 above)
Comment (edwardhartnett) [reply docs/dx_tables.md:54]: done

File: docs/dx_tables.md (hunk 1 above)
Comment (edwardhartnett) [reply docs/dx_tables.md:5]: OK, but that also puts the brief at the top of the documentation page, which looks weird.

I'll leave it for now and if I figure out a better way to deal with it I'll circle back around to it...

File: docs/dx_tables.md [hunk 4]
```diff
@@ -3,14 +3,660 @@
...
+
+<br>
+
+By international convention, a mnemonic should not be given an X value
+between 00 and 47 along with a Y value between 001 and 191 unless that
+mnemonic, when subsequently defined, corresponds exactly to the BUFR
+descriptor having that same FXY number within the [official WMO master
+BUFR tables](@ref wmomstab).
+
+<br>
```
Comment (jbathegit) [inline docs/dx_tables.md:104]: FYI, Doxygen has trouble when you split a hyperlink across multiple lines in a markdown file.  It's best to have the entire `[]()` nomenclature on one line.

Comment (jbathegit) [reply docs/dx_tables.md:104]: But since you've said you're stepping back now, I can go ahead and push up this minor tweak myself and then approve and merge this PR.

**Review by jbathegit — APPROVED** [review]
